extensions/amp-a4a/0.1/callout-vendors.js

```diff
@@ -0,0 +1,5 @@
+/** @const {!Object} */
+export const RTC_VENDORS = {
+  'fakeVendor': 'https://localhost:4443/posts?slot_id=SLOT_ID&page_id=PAGE_ID',
```

Comment:
Nit: these are case-insensitive right?  Perhaps make the clearer here and not use camelcase?

Reply:
done, actually have to change to making it required that all entries in this file be all-lower case for the keys (otherwise we won't be able to just access the urls by key)